cfme/tests/ansible/test_embedded_ansible_crud.py

```diff
@@ -30,15 +27,16 @@ def test_embedded_ansible_enable(embedded_appliance):
         tags: ansible_embed
     """
     assert wait_for(lambda: embedded_appliance.is_embedded_ansible_running, num_sec=30)
-    assert wait_for(lambda: embedded_appliance.supervisord.is_active, num_sec=30)
-    assert wait_for(lambda: embedded_appliance.rabbitmq_server.running, num_sec=30)
-    assert wait_for(lambda: embedded_appliance.nginx.running, num_sec=30)
-    endpoint = "api" if embedded_appliance.is_pod else "ansibleapi"
-
-    assert embedded_appliance.ssh_client.run_command(
-        'curl -kL https://localhost/{endp} | grep "AWX REST API"'.format(endp=endpoint),
-        container=embedded_appliance.ansible_pod_name,
-    )
+    if embedded_appliance.version < "5.11":
+        assert wait_for(lambda: embedded_appliance.supervisord.is_active, num_sec=30)
+        assert wait_for(lambda: embedded_appliance.rabbitmq_server.running, num_sec=30)
+        assert wait_for(lambda: embedded_appliance.nginx.running, num_sec=30)
```

Comment:
OPTIONAL: Don't you need to add `delay` as well? The default value appears to be 1.

Reply:
@valaparthvi It is only checking `service` `status` and returning `True` or `False` So delay is not require there.